### PR TITLE
Support azure storage snapshots

### PR DESCRIPTION
**Pull request by tabossert**
- Support Azure blob storage
- Modify output language
- Fix build error by setting useSSL to bool

### CODE REVIEW

**Review by mjlubrano — APPROVED** [review]
Looks good. Send it!

**Comment by wjimenez5271** [comment]
+1 for Azure support

File: main.go [hunk 1]
```diff
@@ -68,7 +68,9 @@ func CreateSnapshotRepository(elasticURL, repoType, bucketName, username, passwo
 	client := &http.Client{Transport: tr}
 	url := fmt.Sprintf("%s://%s:9200/_snapshot/%s", scheme, elasticURL, bucketName)
 	body := fmt.Sprintf("")
-	if repoAccessKey!="" && repoSecretKey!="" {
+	if repoType=="azure" {
+		body = fmt.Sprintf("{ \"type\": \"%s\", \"settings\": { \"container\": \"%s\", \"compress\": \"true\"  } }", repoType, bucketName)
+	} else if repoAccessKey!="" && repoSecretKey!="" {
```
Comment (stevesloka) [inline main.go]: We should standardize on the check. If we have a flag to determine repo-type, then let's make that consistent.

**Review by stevesloka — CHANGES_REQUESTED** [review]

File: main.go (hunk 1 above)
Comment (tabossert) [reply main.go]: Just pushed a commit, let me know if I understood what you were asking for, if not could you provide a little more direction.  Thanks!

**Comment by tabossert** [comment]
@stevesloka

**Comment by tabossert** [comment]
Ping @stevesloka

**Comment by tabossert** [comment]
Saw you merged the other PR, any chance this could get merged in also soon to support that

**Pull request merged**